w3lib/encoding.py

```diff
@@ -22,14 +22,25 @@ def http_content_type_encoding(content_type):
 
 # regexp for parsing HTTP meta tags
 _TEMPLATE = r'''%s\s*=\s*["']?\s*%s\s*["']?'''
+_SKIP_ATTRS = r'''(?x)(?:\s+
+    [\w-]+  # Attribute name
+    \s*=\s*
```

Comment:
Yes.
I wrote `[\w-]` while looking the possible attribute names of meta
but we are supposed to also deal with human errors.
I'll patch it.